python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -346,6 +354,37 @@ def config_type_pipeline_run(cls, default_image_pull_policy=None):
                 "Default: ``[]``. See: "
                 "https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables",
             ),
+            "volume_mounts": Field(
+                Array(
+                    Shape(
+                        {
+                            "name": str,
+                            "path": str,
```

Comment:
```suggestion
                            "mount_path": str,
```

Also, what about `mount_propogation` and `sub_path_expr`?

Reply:
I think when in doubt we should match the k8s API rather than innovating

Reply:
oh i misunderstood, you meant we should fill out the whole VolumeMount API. Good point